metisp/pymetis/src/pymetis/recipes/ifu/metis_ifu_telluric.py

```diff
@@ -115,15 +114,15 @@ def process(self) -> set[DataItem]:
         combined = self.inputset.combined.load_data('DET1.DATA')
 
         product_telluric_transmission = self.ProductTelluricTransmission(
-            primary_header,
+            header_transmission,
```

Comment:
Should it be the same for the primary and the extensions?

Reply:
Eh no, I think I misunderstood, I'll change it.